internal/pkg/template/workload.go

```diff
@@ -210,7 +210,8 @@ type LogConfigOpts struct {
 
 // HTTPTargetContainer represents the target group of a load balancer that points to a container.
 type HTTPTargetContainer struct {
-	Port string // Port of the container.
+	Container string
```

Comment:
can we rename the field to `Name` since the type has the word `Container` in it 🥺